mirr254/ci-cd-k8s-jenkins
# #160662120 Use terraform to create the infrastructure

#### What does this PR do?
Use terraform to create the infrastructure
#### Description of Task to be completed?
- create a VPC
- Create 3 instances for every worker, controller and etcd
- 
#### How should this be manually tested?
- clone the repo
- checkout to this branch
- change the `default_key_name` in `variables.tf`
- run `terraform plan` to check the infrastructure if satisfied,
- run `terraform apply` to create the infrastructure

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[160662120](https://www.pivotaltracker.com/story/show/160662120)
#### Screenshots (if appropriate)
#### Questions:
